src/main/java/uk/gov/hmcts/reform/prl/controllers/ManageOrdersController.java

```diff
@@ -385,6 +385,9 @@ public AboutToStartOrSubmitCallbackResponse saveOrderDetails(
                 ));
             }
             manageOrderService.setMarkedToServeEmailNotification(caseData, caseDataUpdated);
+            //PRL-4216 - save server order additional documents if any
+            manageOrderService.saveAdditionalOrderDocuments(authorisation, caseData, caseDataUpdated);
```

Comment:
Same call needs to be made in EditAndApproveDraftOrderController aboutToSubmit callback

Reply:
@shashisk97 Done, copied the changes to EditAndApproveDraftOrderController.
I hope all input data fields remains the same, will verify & update.

Reply:
changes are added in EditAndApproveDraftOrderController & verified in PR env.